.env

```diff
@@ -0,0 +1,11 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1671\cocoasubrtf500
```

Comment:
Looks like your rich text editor has added some unexpected and invalid stuff here. That'll need fixing (open it in Atom or something similar).